actionmailer/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+*   Add fallback to template lookup.
```

Comment:
I am not sure if this changelog is actually describing the change or just explaining the implementation details. Can you expend? 

thanks